requirements.txt

```diff
@@ -9,6 +9,7 @@ mypy_extensions~=0.4.3
 typed_ast~=1.4.0
 jsonschema~=3.0.0
 texar-pytorch>=0.1.1
+transformers==3.3.1
```

Comment:
Let's don't put this into the core `requirements`. If the user don't use our DA algorithm, he/she don't have to install transformers. We can simply put this into `setup.py`.